app/views/public/schedule/_events.html.haml

```diff
@@ -7,8 +7,8 @@
       %thead
         %tr
           %th &nbsp;
-          %th= t '.title'
-          %th= t '.speakers'
+          %th= t 'activerecord.attributes.event.title'
```

Comment:
I would have used [human_attribute_name](http://api.rubyonrails.org/classes/ActiveModel/Translation.html#method-i-human_attribute_name):

```ruby
Event.human_attribute_name('title')`
```

Reply:
That way, no need to be aware of how things are organized in i18n files to understand what the code is trying to do :stuck_out_tongue_winking_eye: